cmake/Modules/AXConfigDefine.cmake

```diff
@@ -224,8 +224,8 @@ if(EMSCRIPTEN)
         list(APPEND _AX_EM_LD_FLAGS -pthread -sPTHREAD_POOL_SIZE=${AX_WASM_THREADS})
     endif()
 
-    set(AX_WASM_INITIAL_MEMORY "1024MB" CACHE STRING "")
-    list(APPEND _AX_EM_LD_FLAGS -sINITIAL_MEMORY=${AX_WASM_INITIAL_MEMORY})
+    set(AX_WASM_INITIAL_MEMORY "100MB" CACHE STRING "")
```

Comment:
why 100MB? maybe 128MB?

Reply:
I don't mind putting 128, I'm not sure why a power of two would change much :)